packages/component/src/Transcript/TranscriptFocus/TranscriptFocusArea.tsx

```diff
@@ -0,0 +1,27 @@
+import { useStyles } from '@msinternal/botframework-webchat-styles/react';
+import cx from 'classnames';
+import React, { forwardRef, type HTMLAttributes } from 'react';
+
+import TranscriptFocusIndicator from './TranscriptFocusIndicator';
+
+import styles from './TranscriptFocus.module.css';
+
+type TranscriptFocusAreaProps = HTMLAttributes<HTMLDivElement>;
+
+const TranscriptFocusArea = forwardRef<HTMLDivElement, TranscriptFocusAreaProps>(
+  ({ className, children, ...props }, ref) => {
```

Comment:
Should we use function name and remove `displayName` below?

```suggestion
  function TranscriptFocusArea({ className, children, ...props }, ref) {
```